### PR TITLE
Can 452

### PR DESCRIPTION
**Pull request by shahab-ramzan**
card are now collapse able on topic details page

### CODE REVIEW

File: src/components/ComponentPages/TopicDetails/CampStatementCard/index.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import CustomButton from "@/components/common/button";
-import { Card, Typography, Button } from "antd";
+import { Card, Typography, Button, Collapse } from "antd";
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/CampStatementCard/index.tsx]: Probably `Card` is not used.

File: src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Card, Checkbox, Typography } from "antd";
+import { Card, Checkbox, Typography, Collapse } from "antd";
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx]: `Card` and `Typography` are not used.

File: src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx [hunk 3]
```diff
@@ -28,38 +30,40 @@ const CampTreeCard = ({ scrollToCampStatement, getSelectedNode }) => {
...
-      extra={
-        <>
+    <Collapse defaultActiveKey={['1']} expandIconPosition="right" className="topicDetailsCollapse">
+      <Panel header={<h3>Canonizer Sorted Camp Tree</h3>} 
+        key="1" extra={
+          <>
+          <div className="ant-checkbox-wrapper" onClick={event => {event.stopPropagation();}}>
           <Checkbox
             checked={
               currentCampRecord && currentCampRecord[0].campSubscriptionId
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx]: This probably replaced as `!!(currentCampRecord && currentCampRecord[0].campSubscriptionId)`

File: src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx [hunk 4]
```diff
@@ -1,37 +1,36 @@
-import { Card, Button, Descriptions } from "antd";
+import { Card, Button, Descriptions, Collapse } from "antd";
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx]: `Card` is not used

File: src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx [hunk 5]
```diff
@@ -1,37 +1,36 @@
...
   return (
-    <Card
-      className="canCard mb-3"
-      title={<h3>Current Camp Record</h3>}
-      actions={[
-        <>
+    <Collapse accordion={true} defaultActiveKey={['1']} expandIconPosition="right" className="topicDetailsCollapse">
+      <Panel header={<h3>Current Topic Record</h3>} key="1">
+        <Descriptions column={1}>
+          <Descriptions.Item label="Camp Name">
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx]: May be the associate the array somewhere in the config file and iterate it here.

File: src/components/ComponentPages/TopicDetails/index copy.tsx [hunk 6]
```diff
@@ -0,0 +1,250 @@
+import { Typography } from "antd";
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/index copy.tsx]: Is this file required ? I see `index copy.tsx` in the filename

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx (hunk 2 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx]: fixed

File: src/components/ComponentPages/TopicDetails/CampStatementCard/index.tsx (hunk 1 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/CampStatementCard/index.tsx]: fixed

File: src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx (hunk 3 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx]: we don't need checkbox here, so it will be removed in next  PR

File: src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx (hunk 4 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx]: removed

File: src/components/ComponentPages/TopicDetails/index copy.tsx (hunk 6 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/index copy.tsx]: removed

File: src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx (hunk 5 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx]: associated array added

**Review by akuks — APPROVED** [review]
